locales/templates/app.pot

```diff
@@ -305,6 +305,140 @@ msgctxt "preference_about"
+msgctxt "preference_search_add"
+msgid "Add another search engine"
+msgstr ""
+
+msgctxt "preference_search_remove"
+msgid "Remove"
+msgstr ""
+
+#. Clickable text in tutorial to open a dialog to manually add a search engine
+msgctxt "tutorial_search_manually_add"
+msgid "MANUALLY ADD"
+msgstr ""
+
+msgctxt "tutorial_search_cancel"
+msgid "CANCEL"
+msgstr ""
+
+msgctxt "tutorial_search_title"
+msgid "Add search engine"
+msgstr ""
+
+msgctxt "tutorial_search_step1"
+msgid "1. Go to the search engine homepage."
+msgstr ""
+
+msgctxt "tutorial_search_step2"
+msgid "2. Tap into the search engine's search field."
+msgstr ""
+
+#. An alternative string for Step 2 of the tutorial
+msgctxt "tutorial_search_step2_alternative"
+msgid "2. Long press the search engine's search field."
+msgstr ""
+
+msgctxt "tutorial_search_step3"
+msgid ""
+"3. Tap the magnifying glass icon above the keyboard to add the search "
+"engine."
+msgstr ""
+
+#. Clickable action text that shows when user long-presses on a search engine
+#. search field (this is similar to paste or copy when long-pressing a text
+#. field)
+msgctxt "action_option_add_search_engine"
+msgid "ADD SEARCH ENGINE"
+msgstr ""
+
+#. An alternative string for Step 3 of the tutorial. Parameter 1 is
+#. &action_option_add_search_engine, which is the action text that the tutorial
+#. is referring to
+#, c-format
+msgctxt "tutorial_search_step3_alternative"
+msgid "3. Tap %1$s"
+msgstr ""
+
+msgctxt "tutorial_search_step4"
+msgid ""
+"4. If you don’t see the magnifying glass, try manually adding a search "
+"engine."
+msgstr ""
+
+#. An alternative string for Step 4 of tutorial. Instruction with suggested
+#. action to add search engine, and alternative if that fails. Parameter 1 is
+#. &action_option_add_search_engine, which is the clickable action text for
+#. adding a search engine.
+#, c-format
+msgctxt "tutorial_search_step4_alternative"
+msgid "4. If you don’t see %1$s, try manually adding a search engine."
+msgstr ""
+
+msgctxt "tutorial_search_finish"
+msgid "GET STARTED"
+msgstr ""
+
+msgctxt "search_add_hint"
+msgid ""
+"Tap the magnifying glass icon above the keyboard to add this search "
+"engine."
+msgstr ""
+
+msgctxt "search_add_manually_name"
+msgid "Name to display"
+msgstr ""
+
+msgctxt "search_add_manually_name_hint"
+msgid "Search engine name"
+msgstr ""
+
+msgctxt "search_add_manually_string"
+msgid "Search string to use"
+msgstr ""
+
+msgctxt "search_add_manually_save"
+msgid "Save"
+msgstr ""
+
+#. Explanation of how to add search string, with instruction on how to modify
+#. the string if necessary. The string "%s" should not be changed or removed.
+#, c-format, python-format
+msgctxt "search_add_manually_string_hint"
+msgid "Paste or enter search string. If necessary, replace search term with: %s."
+msgstr ""
+
+#. Example of search engine url
+#, c-format, python-format
+msgctxt "search_add_manually_example"
+msgid "Example: searchengine.com/search/?q=%s"
```

Comment:
This looks like a terrible idea… You're promoting an actual website, that has a Google Custom search engine.

Reply:
example.org?

Reply:
That, or example.com. If you really want a working query URL, I'd suggest to go with something that we own, e.g. https://bugzilla.mozilla.org/show_bug.cgi?id=%s

Reply:
Great, I will update this to example.com